server/middleware/validationMiddleware.ts

```diff
@@ -39,8 +39,10 @@ function validationMiddleware(type: new () => object): RequestHandler {
     const flattenErrors: any = (errorList: ValidationError[], parent?: string) => {
       // Flat pack a list of errors with child errors into a 1-dimensional list of errors.
       return errorList.flatMap(error => {
+        const property = `${parent ? `${parent}-` : ''}${error.property}`
```

Comment:
Possibly add tests to cover these changes to explain them? I believe they're to deal with the nested validation on the new bulk appointment times page